githubapi_class.inc

```diff
@@ -380,10 +381,6 @@ class GitHubAPI {
   public function getRequest($command, $params = array(), $headers = array()) {
     $this->prepareCommand($command, $params);
 
-    if ($this->getToken()) {
-      $params['access_token'] = $this->getToken();
-    }
```

Comment:
Could you also update putRequest() and postRequest() (and any other places) that still use the `access_token` parameter?